synapse/models/biz.py

```diff
@@ -0,0 +1,204 @@
+                }),
+                ('biz:deal', ('guid', {}), {
+                    'doc': 'A sales or procurement effort in pursuit of a purchase.',
+                }),
+                ('biz:bundle', ('guid', {}), {
+                    'doc': 'Instances of a specific product offered for a price.',
+                }),
+                ('biz:product', ('guid', {}), {
+                    'doc': 'A product which is available for purchase.',
+                }),
+                ('biz:dealstatus', ('taxonomy', {}), {
+                    'doc': 'A deal/rfp status taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+                ('biz:dealtype', ('taxonomy', {}), {
+                    'doc': 'A deal type taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+                ('biz:prodtype', ('taxonomy', {}), {
+                    'doc': 'A product type taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+            ),
+            'forms': (
+                ('biz:dealtype', {}, ()),
+                ('biz:prodtype', {}, ()),
+                ('biz:dealstatus', {}, ()),
+                ('biz:rfp', {}, (
+                    ('ext:id', ('str', {}), {
+                        'doc': 'An externally specified identifier for the RFP.',
+                    }),
+                    ('title', ('str', {}), {
+                        'doc': 'The title of the RFP.',
+                    }),
+                    ('summary', ('str', {}), {
+                        'disp': {'hint': 'text'},
+                        'doc': 'A brief summary of the RFP.',
+                    }),
+                    ('status', ('biz:dealstatus', {}), {
+                        'disp': {'hint': 'enum'},
+                        'doc': 'The status of the RFP.',
+                    }),
+                    ('url', ('inet:url', {}), {
+                        'doc': 'The official URL for the RFP.',
+                    }),
+                    ('posted', ('time', {}), {
+                        'doc': 'The date/time that the RFP was posted.',
+                    }),
+                    ('quesdue', ('time', {}), {
+                        'doc': 'The date/time that questions are due.',
+                    }),
+                    ('propdue', ('time', {}), {
+                        'doc': 'The date/time that proposals are due.',
+                    }),
+                    ('contact', ('ps:contact', {}), {
+                        'doc': 'The contact information given for the org reqesting offers.',
+                    }),
+                    ('purchases', ('econ:purchase', {}), {
+                        'doc': 'Any known purchases that resulted from the RFP.',
+                    }),
+                    ('requirements', ('array', {'type': 'ou:goal'}), {}),
+                )),
+                ('biz:deal', {}, (
```

Comment:
Curious of the diff b/t biz:deal and ou:contract? And why we opted to created this form vs extending the contract node?

Reply:
Happy to discuss, but it's a classic "potential vs actual" model issue, as well as a different set of tracking characteristics.